configure.ac

```diff
@@ -209,9 +209,21 @@ AC_CONFIG_FILES([Makefile
                  data/pixmaps/Makefile
                  tests/Makefile
                  ui/Makefile
-                 ui/webui/Makefile
                  utils/Makefile
                  utils/dd/Makefile])
+
+### Include Cockpit part to the make execution as optional ###
+# The main reason to have this is to speed up execution of the unit-tests by avoiding npm package download.
+AC_ARG_ENABLE([cockpit], AS_HELP_STRING([--disable-cockpit], [Build without the cockpit ui; makes the build much faster]))
```

Comment:
s/cockpit/webui

Indeed I have used cockpit in the spec file, but that's rather an exception. Let's use webui everywhere I suggest.